### DISCUSSION TITLE
Cannot connect to Server

### DISCUSSION THREAD

**Discussion by thedmo** [design1online/WTF-Adventure#62] [Q&A]
Hello there

I'm trying to get this running on localhost 

Erverything is setup as mentioned in the Readme:
- installed MAMP

- client config.json:
    - ip: localhost
    - port: 8081

- server config.json:
    - host: localhost
    - port: 8081
    - sqldatabase is setup on localhost
        -  entries are being made when is run wtfserver so comm with database seems to work.

After npm start, the startpage opens and i'm on the login page. On "guest login" i'm getting the Message "Could not connect to the game server". Console output from the Browser shows that it can't connect to the websocket (see attached picture)

what do i miss?

![grafik](https://user-images.githubusercontent.com/17425466/108355020-8a35ce00-71ea-11eb-8f4d-070fb2040746.png)

**Comment by Bartolomeus76** [comment]
Didn´t work, Server running without Errors, MySQL-Connection OK but Website only shows "CANNOT GET /" on white screen.
Firefox Console showing no error. Because there are no Logs for server there is no chance to specify the problem...
